Source/Scene/Cesium3DTileset.js

```diff
@@ -226,7 +226,7 @@ define([
         this._stencilClearCommand = undefined;
         this._backfaceCommands = new ManagedArray();
 
-        this._maximumScreenSpaceError = defaultValue(options.maximumScreenSpaceError, 16);
+        this._maximumScreenSpaceError = defaultValue(options.maximumScreenSpaceError, 16 * window.devicePixelRatio);
```

Comment:
I'm not totally sure about this change. If someone wants to change the maximum SSE they will need to remember to multiply by the `window.devicePixelRatio` to get the expected behavior. And existing apps that set an explicit maximum SSE will start to load more tiles unless they make that change.

Instead maybe there should be a property in `Scene` (and passed through to `FrameState`) called `screenSpaceErrorTargetsPhysicsUnits` that defaults to `true`. When `true` it does the behavior in this PR but is completely opaque to the user, so `maximumScreenSpaceError` would default to 16 but some internal property like `_maximumScreenSpaceError` would be `maximumScreenSpaceError * windows.devicePixelRatio` and would be used for the actual traversal. When `false`, `_maximumScreenSpaceError` would not be multiplied by anything.

If you can think of a clearer name for `screenSpaceErrorTargetsPhysicsUnits` though that would be good.

Also I think it should multiply by `resolutionScale` instead of `window.devicePixelRatio`.

Reply:
Also sorry for this last minute suggestion, it dawned on me as I was doing my final look at the code.

Reply:
This is a good catch @lilleyse I agree with both of your comments and feel bad for missing it.  Not sure about the name, but that's easy to change once the functionality is fixed. @ProjectBarks please let us know if you have any questions.

Reply:
Here are some names I came up with:
`sseResolutionDependent`
`useResolutionSSEMultiplier`
`useResolutionScaledScreenSpaceError`

Reply:
Talked to Sean offline and we came to the conclusion that we do not need a separate properly to enable or disable scaling SSE by `resolutionScale`. The rationale was the SSE should work independently of `resolutionScale`. The idea that would want a change in resolution scale effecting how their models load seems antithetical to common expectation. If the demand is there we can always add the property in, but for now we are going to proceed without a toggle.

Reply:
But this isn't `resolutionScale`, it's `devicePixelRatio`.  The latter is affected by things such as `CTRL +` and `CTRL -` screen scaling, and Windows Desktop DPI scaling.  Just for completeness, has someone tested the effects of those being set to non-default settings?

Reply:
`resolutionScale` used to be a constant 1 that could be changed by the user but now it's set to `window.devicePixelRatio` by default. 

`viewer.resolutionScale = 1`
<img width="314" alt="image" src="https://user-images.githubusercontent.com/4194371/63177191-643bd600-c015-11e9-9cc4-341f65b53bfb.png">

`viewer.resolutionScale = 2`
<img width="271" alt="image" src="https://user-images.githubusercontent.com/4194371/63177083-3060b080-c015-11e9-8583-3c8530faf713.png">

`viewer.resolutionScale = 2` & `maximumScreenSpaceError: 16 * viewer.resolutionScale`
<img width="319" alt="image" src="https://user-images.githubusercontent.com/4194371/63177306-a2d19080-c015-11e9-9fb8-c8107f0b6d46.png">

Reply:
So maybe there should be an option to keep matching resolution scale to the browsers resolution scale (ie fixing zoom in and zoom out). But otherwise this fixes the inadvertent increase of tile load when resolution increases.

Reply:
@emackey would it make sense to do `_maximumScreenSpaceError = maximumScreenSpaceError * resolutionScale * devicePixelRatio`? Also see https://github.com/AnalyticalGraphicsInc/cesium/pull/8057#discussion_r314772801 below.